fastlane/Fastfile

```
@@ -0,0 +1,115 @@
+# Common lane to bump version in package.json
+desc "Bump package.json version and return new version"
+lane :bump_package_json_version do
+  require 'json'
+
+  package_json_path = File.expand_path("../package.json", __dir__)
+  unless File.exist?(package_json_path)
+    UI.user_error!("Error: package.json not found at #{package_json_path}")
+  end
+
+  package_json = JSON.parse(File.read(package_json_path))
+  current_version = package_json["version"]
+
+  if current_version.nil? || current_version.empty?
+    UI.user_error!("Error: Version not found in package.json")
+  end
+
+  major, minor, patch = current_version.split('.').map(&:to_i)
```

Edge case: if `package.json` has a pre-release (e.g., `1.2.3-beta.1`) or non-3-part version, this will crash. Suggest validating with a regex and failing fast with a friendly error, or normalizing to `x.y.z`.

---

> Edge case: if `package.json` has a pre-release (e.g., `1.2.3-beta.1`) or non-3-part version, this will crash. Suggest validating with a regex and failing fast with a friendly error, or normalizing to `x.y.z`.

Added via f7a1ed02906bc57012ebffe2aa7d7c7b24b8495d